examples/train.rs

```diff
@@ -44,14 +44,14 @@ fn main() {
         )
     };
 
-    for i_epoch in 0..1 {
+    for i_epoch in 0..10 {
         for (img, lbl) in train_data
             .shuffled(&mut rng)
             .map(preprocess)
             .batch(batch)
             .collate()
             .stack()
-            .take(30)
+            .take(128)
```

Comment:
Have you tried using the whole dataset? I had the `.take()` in there to make profiling easier, but really we'd want to use the whole dataset for training

Reply:
Ah, good point. Just ran it with both setting `.take(50_000)` and `// take()` (commenting it out entirely), both of which improve the results up to about 40%, but not more; training epochs are still set to 10.

Reply:
Gotcha - the other thing is I kind of arbitrarily set up the small resnet architecture, I have no idea if it actually works haha. Might be worth trying to run it in pytorch to see if it achieves a different performance. That would give us a clue if there's a bug somewhere or just the network structure is bad